usage.md

```diff
@@ -8,45 +8,43 @@ Installation will be required for capabilities such as [self-provisioning](doc/d
 
 ### Requirements
 
-The first three requirements derive from a placeholder install script. These will be addressed in #121.
+1. ESXi/vCenter - the target virtualization environment
```

Comment:
We don't need to mention the kernel - it's not a requirement on environment the user supplies.
I do mean ESXi/vCenter configuration - I'm fine with something that simply says only simple configurations are supported for now.

Two examples of specific configs that may be a problem currently:
Enhanced Linked Mode
Datastore Clusters/Storage vMotion

I'm not asking that we list everything that won't work, rather that we note the general configuration that is known to work at this time.